README.md

```diff
@@ -160,6 +162,12 @@ The following interfaces can have custom implementations registered via [init()]
 [database/sql]: https://godoc.org/database/sql
 [main.go]: https://github.com/coreos/clair/blob/master/cmd/clair/main.go
 
+## Screencast
```

Comment:
Maybe this should be a subsection of "Hello Heartbleed" closer to the docker-compose instructions?